ehb_service/apps/core/models/identities.py

```diff
@@ -197,7 +196,8 @@ class Group(CreatedModified):
     client_key = models.CharField(max_length=255, verbose_name='Client KEY', editable=True)
     is_locking = models.BooleanField(default=False, verbose_name='Lock Group')
     # this cannot use onetoone because I don't want the ehb_keys to EVER be deleted
-    ehb_key = models.ForeignKey(GroupEhbKey, editable=False, blank=True, unique=True)
+    # ehb_key = models.OneToOneField(GroupEhbKey, editable=False, blank=True)
```

Comment:
Is this a real comment? or can it be deleted?

Reply:
removed: 70b94dc0577eeb5e2c319a0ef740a793e126e937